src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/util/OcrDataDeserializerTest.java

```diff
@@ -33,6 +33,20 @@ public void should_convert_all_fields_to_their_string_representation() throws Ex
         assertThat(result).isEqualTo(expectedResult);
     }
 
+    @Test
+    public void should_throw_illega_state_exception_when_ocr_data_contains_duplicate_keys() {
+        Throwable thrown = catchThrowable(() -> {
+                JsonParser jsonParser = getJsonParser("/ocr-data/invalid/invalid-ocr.json");
+                deserializer.deserialize(jsonParser, mock(DeserializationContext.class));
+            }
+        );
+
+        assertThat(thrown).isInstanceOf(OcrDataParseException.class)
+            .hasCauseExactlyInstanceOf(IllegalStateException.class);
+
+        assertThat(thrown.getCause().getMessage()).contains("Duplicate key for value 'some text'");
```

Comment:
The duplicate key is `text_field`.
That would be much more useful information

Reply:
If those are field values then we should just exclude them from the message. They might be sensitive information and we have to be sure it's not logged anywhere, etc..

Reply:
I've updated the log message to hide the values. It just says, there are duplicate fields in the ocr data